scripts/setup_gemdaq.sh

```diff
@@ -207,41 +288,58 @@ fi
-then
-    export DATA_PATH=/afs/cern.ch/work/$USER[0,1]/$USER/CMS_GEM/Data/gemdata
-elif [[ $SYSTEM_INFO == *"gem904"* ]];
+if [[ -z $DATA_PATH ]]; # Don't override existing value
 then
-    export DATA_PATH=/data/bigdisk/GEM-Data-Taking/GE11_QC8/
-    export LD_LIBRARY_PATH=/opt/cactus/lib:$LD_LIBRARY_PATH
-    export GEM_ADDRESS_TABLE_PATH=/opt/cmsgemos/etc/maps
-    export AMC13_ADDRESS_TABLE_PATH=/opt/cactus/etc/amc13/
+    if [[ $SYSTEM_INFO == *"lxplus"* ]];
+    then
+        export DATA_PATH=/afs/cern.ch/work/${USER:0:1}/$USER/CMS_GEM/Data/gemdata
+    elif [[ $SYSTEM_INFO == *"gem904"* ]];
+    then
+        export DATA_PATH=/data/bigdisk/GEM-Data-Taking/GE11_QC8/
+        export LD_LIBRARY_PATH=/opt/cactus/lib:$LD_LIBRARY_PATH
+        export GEM_ADDRESS_TABLE_PATH=/opt/cmsgemos/etc/maps
+        export AMC13_ADDRESS_TABLE_PATH=/opt/cactus/etc/amc13/
 
-    # Firmware
-    export FIRMWARE_GEM=/data/bigdisk/GEMDAQ_Documentation/system/firmware/files
+        # Firmware
+        export FIRMWARE_GEM=/data/bigdisk/GEMDAQ_Documentation/system/firmware/files
 
-    # AMC13 Tool
-    alias AMC13Tool2.ext='/opt/cactus/bin/amc13/AMC13Tool2.exe'
-    alias AMC13ToolQC8='/opt/cactus/bin/amc13/AMC13Tool2.exe -c  192.168.2.104/c'
+        # AMC13 Tool
+        alias AMC13Tool2.ext='/opt/cactus/bin/amc13/AMC13Tool2.exe'
+        alias AMC13ToolQC8='/opt/cactus/bin/amc13/AMC13Tool2.exe -c  192.168.2.104/c'
 
-    # xDAQ
-    alias xdaq=/opt/xdaq/bin/xdaq.exe
+        # xDAQ
+        alias xdaq=/opt/xdaq/bin/xdaq.exe
 
-    # Misc
-    #alias arp-scan='sudo /usr/sbin/arp-scan'
-    alias gbtProgrammer='java -jar /data/bigdisk/sw/GBTx_programmer/programmerv2.20180116.jar'
-elif [[ $SYSTEM_INFO == *"srv-s2g18"* || $SYSTEM_INFO == *"kvm"* ]];
-then
-    export DATA_PATH=/gemdata
-    export LD_LIBRARY_PATH=/opt/cactus/lib:$LD_LIBRARY_PATH
+        # Misc
+        #alias arp-scan='sudo /usr/sbin/arp-scan'
+        alias gbtProgrammer='java -jar /data/bigdisk/sw/GBTx_programmer/programmerv2.20180116.jar'
+    elif [[ $SYSTEM_INFO == *"srv-s2g18"* || $SYSTEM_INFO == *"kvm"* ]];
+    then
+        export DATA_PATH=/gemdata
+        export LD_LIBRARY_PATH=/opt/cactus/lib:$LD_LIBRARY_PATH
+    fi
 fi
 
 # Setup path
 export PATH=$VENV_DIR/lib/python$PYTHON_VERSION/site-packages/gempython/scripts:$PATH
 export PATH=$VENV_DIR/lib/python$PYTHON_VERSION/site-packages/gempython/gemplotting/macros:$PATH
 
 # Create mapping files
-if [ ! -f $VENV_DIR/lib/python2.7/site-packages/gempython/gemplotting/mapping/shortChannelMap.txt ]
+if [ ! -f $VENV_DIR/lib/python$PYTHON_VERSION/site-packages/gempython/gemplotting/mapping/shortChannelMap.txt ]
 then
     find $VENV_DIR/lib/python$PYTHON_VERSION/site-packages/gempython -type f -name buildMapFiles.py -exec python {} \;
 fi
+
+if [ ! -d "$DATA_PATH" ];
+then
+    echo "WARNING: The directory \"$DATA_PATH\" (\$DATA_PATH) doesn't exist."
+    echo "         Fix your \$DATA_PATH using export DATA_PATH=..."
+fi
```

Comment:
Remove lines 333-337.  See discussion [here](https://github.com/cms-gem-daq-project/sw_utils/pull/1/files#r193545436).  For 904 & p5 this will always exist and will be the centrally defined location; for `lxplus` it will be made if it does not exist.